File: Butkemp19/Department.cs
class Department
{
    public int id;
    public string tittle;

    public Department(int id, string tittle)
    {
        this.id = id;
        this.tittle = tittle;
    }
    public override string ToString()
    {
        return $"tittle: {this.tittle} id: {this.id}";
    }
}

File: Butkemp19/Department.py
class Department:
    id: int
    tittle: str

    def __init__(self, id: int, title: str) -> None:
        self.id = id
        self.tittle = title

    def __repr__(self) -> str:
        return f"title: {self.tittle} id: {self.id}"